api/src/test/java/com/epam/pipeline/manager/pipeline/FolderTemplateManagerTest.java

```diff
@@ -75,98 +67,130 @@
     private static final String CHILD_TEMPLATE_FOLDER_NAME_1 = "test-1";
     private static final String DATASTORAGE_NAME_1 = "ds-1";
 
+    private static final Long FOLDER_ID = 1L;
+    private static final Long FOLDER_CHILD_ID = 2L;
+
     @Autowired
     private FolderTemplateManager folderTemplateManager;
     @Autowired
-    private FolderManager folderManager;
-    @Autowired
-    private MetadataManager metadataManager;
-    @Autowired
-    private DataStorageManager dataStorageManager;
-    @SpyBean
-    private S3StorageProvider storageProviderManager;
+    private MetadataManager mockMetadataManager;
     @Autowired
-    private AclTestDao aclTestDao;
+    private DataStorageManager mockDataStorageManager;
     @Autowired
-    private GrantPermissionManager permissionManager;
+    private GrantPermissionManager spyGrantPermissionManager;
     @Autowired
-    private CloudRegionDao cloudRegionDao;
+    private FolderCrudManager mockFolderCrudManager;
 
-    private AwsRegion awsRegion;
+    private Map<String, PipeConfValue> metadata;
+    private PermissionVO permissionVO;
+    private FolderTemplate folderTemplate;
 
+    @Test
+    public void createFolderFromTemplateTest() throws IOException {
+        initializeParameters();
+        preConditions();
```

Comment:
I would suggest to inline both preambule and verification methods for now. It doesn't really help increase readability but  introduces several fields rather than local variables.

Later these parameters initialization can be simplified using utils from `com.epam.pipeline.test.creator` package.

Reply:
Fixed